api/tools/src/main/java/org/pmiops/workbench/tools/ManageLeonardoRuntimes.java

```diff
@@ -163,17 +164,12 @@ private static void describeRuntime(
     // Leo's getRuntime API swagger tends to be outdated; issue a raw getRuntime request to ensure
     // we get all available information for debugging.
     RuntimesApi client = newApiClient(apiUrl);
-    com.squareup.okhttp.Call call =
-        client.getRuntimeCall(
-            googleProject,
-            runtimeName,
-            /* progressListener */ null,
-            /* progressRequestListener */ null);
```

Comment:
why this change?

Reply:
it's just the generated client has a dependency change..so the only change here is the actual type declaration that's returned by `client.getRuntimeCal`